docs/src/cloud/cloud_faq.md

```diff
@@ -0,0 +1,34 @@
+This section provides a set of recommended best practices to help you get the most out of LanceDB Cloud. By following these guidelines, you can optimize your usage of LanceDB Cloud, improve performance, and ensure a smooth experience.
+
+### Should the db connection be created once and keep it open?
+Yes! It is recommended to establish a single db connection and maintain it throughout your interaction with the tables within. 
+
+LanceDB uses HTTP connections to communicate with the servers. By re-using the Connection object, you avoid the overhead of repeatedly establishing HTTP connections, significantly improving efficiency.
+
+### Should a single `open_table` call be made and maintained for subsequent table operations?
```

Comment:
```suggestion
### Should I re-use the `Table` object?
```

Reply:
Another related question: "Is there any cleanup I need to do when closing a Connection / Table". And the answer is no, those both have destructors that handle closing connections.